### ISSUE TITLE
EntryPointNotFoundException: SteamAPI_ISteamUtils_IsSteamRunningOnSteamDeck assembly:<unknown assembly> type:<unknown type> member:(null)

### ISSUE THREAD

**Issue by 3ternal** [rlabrecque/Steamworks.NET#589]
Hi,

I'm getting an error when I call the `SteamUtils.IsSteamRunningOnSteamDeck()` method after initializing Steam (i.e. after SteamManager.Initialized returns true).

```
EntryPointNotFoundException: SteamAPI_ISteamUtils_IsSteamRunningOnSteamDeck assembly:<unknown assembly> type:<unknown type> member:(null)
Steamworks.SteamUtils.IsSteamRunningOnSteamDeck () (at ./Library/PackageCache/com.rlabrecque.steamworks.net@a3fe8091f7/Runtime/autogen/isteamutils.cs:326)
```

Unity 2022.3.6
Steamworks.NET 20.2.0
Error occurs in both editor and build.

**Comment by EclipseStorm** [comment]
I'm having the same issue.
Are there any known fixes?

**Comment by 3ternal** [comment]
> I'm having the same issue. Are there any known fixes?

I'm not sure either, unfortunately. I commented it out when I encountered the bug, but hopefully there's some fix in the future. Haven't checked if it works on the Facepunch Steamworks package.

**Comment by 3ternal** [comment]
Actually, I just solved this while troubleshooting something else. It's similar to the problem described here: https://github.com/rlabrecque/Steamworks.NET/issues/558

I fixed it by deleting the leftover Steam-related DLLs from `Plugins/x86_64` and `Plugins/x86`. Those DLLs are currently in the immutable Packages folder, so we don't want duplicates lying around the Assets folder.

I also grabbed the latest Steamworks SDK from Valve itself (separate from Unity): https://partner.steamgames.com/downloads/list

Not sure if that was related, but I forgot that we're supposed to do that every now and then.
